### PR TITLE
Allow Pool-level Rewards distributors in Perps

### PR DESCRIPTION
**Pull request by noisekit**

### CODE REVIEW

**Review by dbeal-eth — APPROVED** [review]
all we need to do is add a test "can add RD at zero address" or whatever

File: markets/perps-market/contracts/storage/LiquidationAssetManager.sol [hunk 1]
```diff
@@ -53,9 +53,6 @@ library LiquidationAssetManager {
             revert ParameterError.InvalidParameter("collateralTypes", "must not be empty");
         }
         for (uint256 i = 0; i < collateralTypesLength; ) {
-            if (self.poolDelegatedCollateralTypes[i] == address(0)) {
-                revert AddressError.ZeroAddress();
```
Comment (noisekit) [inline markets/perps-market/contracts/storage/LiquidationAssetManager.sol:57]: later in the code there is an iteration over all the collaterals configured and then their prices are used. So we end up having error that Zero-address oracle node is used.

Comment (noisekit) [reply markets/perps-market/contracts/storage/LiquidationAssetManager.sol:57]: so to use pool level distributors with perps we would need quite a bit more refactoring

**Review by kaleb-keny — CHANGES_REQUESTED** [review]
Blocked until issues resolved
